### ISSUE TITLE
CI builds could be better labeled

### ISSUE THREAD

**Issue by RetroEdit** [nba-emu/NanoBoyAdvance#311]
Ideally, development builds would have the commit hash in filename of the artifact and also in the `Help > About NanoBoyAdvance` menu. The `About` page could also have the NBA version for that matter, though I guess that's in the window title. As far as I'm aware, this information is missing from the downloaded files.

I guess there's an alternative download that gives the build ID, but that's still a bit indirect.

**Comment by fleroviux** [comment]
Yeah, the way this is handled currently definitely isn't ideal. The emulator version is also hardcoded in a couple of places currently.

**Comment by fleroviux** [comment]
![image](https://github.com/nba-emu/NanoBoyAdvance/assets/13669774/7be806fc-2b2b-49bb-ae8f-c75e4bf5ac9a)

In development builds the window title now includes the branch name and commit hash.

**Comment by RetroEdit** [comment]
Including the version + branch name + hash in `Help > About NanoBoyAdvance` would also be useful for copy-pasting, but this is definitely an improvement.
